CONTRIBUTING.md

```diff
@@ -22,9 +22,15 @@ facilitates social contribution, easy testing and peer review.
 
 To contribute a patch, the workflow is as follows:
 
-  - Fork repository
-  - Create topic branch
-  - Commit patches
```

Comment:
Would it be worth enabling Github's Discussion feature for a complex topic and add it here as a good practice to review the issue before creating PR? This could avoid devs wasting time developing something that goes against the roadmap plan—just a suggestion.

Reply:
Others have suggested this, I haven't had time to look into it yet, but I think it's on the to-do list

Reply:
Suggested in #2128 exactly.